### ISSUE TITLE
Support more Japanese naming conventions 

### ISSUE THREAD

**Issue by kanjieater** [Snd-R/komf#74]
Japanese series come in a number of naming formats. There are really only a few patterns that need to be accounted for as far as a normalized library would be concerned though. Here are the ones that would be worth supporting. This is from a config file for ComicEater. The patterns are in priority order top to bottom. The `pattern` is only selected if all the available `{{` templates have content. 

I am assuming Komf uses the file system's file names not the Komga titles (sometimes I customize those).

```yml
  outputNamingConventions:
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeRange}}巻"
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeNumber}}{{volumeVariant}}巻"
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeNumber}}.{{subChapter}}話"
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeNumber}}巻"
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeNumber}}話"
  - "{{seriesRoot}}{{seriesName}}/{{seriesName}} - 第{{volumeNumber}}.{{subChapter}}話"
  - "{{seriesRoot}}{{seriesName}}/{{seriesName}} - 第{{volumeRange}}巻"
  - "{{seriesRoot}}{{seriesName}}/{{seriesName}} - 第{{volumeNumber}}{{volumeVariant}}巻"
  - "{{seriesRoot}}{{seriesName}}/{{seriesName}} - 第{{volumeNumber}}巻"
  - "{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}}"
```

Example file name:

`[冬虫カイコ] みなそこにて - 第01巻` `"{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeNumber}}巻"`

The last character is 巻 for volumes, and 話 for chapters.
Sometimes manga uses metadata next to the numbers {{volumeNumber}}{{volumeVariant}}

`[冬虫カイコ] みなそこにて - 第01w巻`, where a single character might come after the volume or volume range

Sometimes multiple volumes are packed together: `[冬虫カイコ] みなそこにて - 第01-09巻`, so this would match `"{{seriesRoot}}{{seriesName}}/[{{authors}}] {{seriesName}} - 第{{volumeRange}}巻"`

If we don't know the author, it wouldn't be in the brackets. `みなそこにて - 第10巻` if we do, then it would `[冬虫カイコ] みなそこにて - 第10巻`

TLDR;
While there's only a few other patterns that should be considered, this naming convention is pretty much the standard;
`みなそこにて - 第10巻` and `[冬虫カイコ] みなそこにて - 第10巻`
